RevenueCatUI/Purchasing/PurchaseHandler.swift

```diff
@@ -40,6 +59,12 @@ final class PurchaseHandler: ObservableObject {
     @Published
     fileprivate(set) var purchaseResult: PurchaseResultData?
 
+    @Published
+    fileprivate(set) var handlePurchase: HandlePurchaseData?
+
+    @Published
+    fileprivate(set) var handleRestore: HandleRestoreCallbackContainer?
+
```

Comment:
Mentioned in Slack but it would be neat if we could combine these in one view modifier because both need to be implemented